lib/config_upgrader.js

```diff
@@ -269,7 +269,11 @@ class ConfigUpgrader {
       return report;
     });
 
-    return reports.reduce(function(a, b) { return a.concat([""]).concat(b); });
+    if (reports.length > 0) {
```

Comment:
I like the explicitness of `> 0` here. The fact that `0` is falsy makes me squirm heh